client/src/components/ReviewsList/index.jsx

```diff
@@ -220,53 +263,89 @@ class ReviewsList extends React.Component {
     }
 
     return (
-      <div className={styles.reviewsList}>
-        <div>
-          <div className={styles.ratingsOverview}>
-            <h3 className={styles.ratingsAndReviewsTitle}>Reviews and Ratings</h3>
-            <div className={styles.overallRating}>{starRating}</div>
-            <div className={styles.starRating}>
-              <ReviewsAverageOverviewStars starRating={starRating} />
-            </div>
-            <div className={styles.recommendOverview}>
-              <div className={styles.recommendedPercent}>
-                {recommendPercent}
-                %
+      <div>
+        <div className={styles.sort}>
+          <div> </div>
+          <div>
+            <button
+              className={styles.newestButton}
+              type="submit"
+              onClick={this.handleSort}
+              value="newest"
+            >
+              Newest
+            </button>
+            <button
+              className={styles.relevantButton}
+              type="submit"
+              onClick={this.handleSort}
+              value="relevant"
+            >
+              Relevant
+            </button>
+            <button
+              className={styles.helpfulButton}
+              type="submit"
+              onClick={this.handleSort}
+              value="helpful"
+            >
+              Helpful
+            </button>
+          </div>
+        </div>
+        <div className={styles.reviewsList}>
+          <div>
+            <div className={styles.ratingsOverview}>
+              <h3 className={styles.ratingsAndReviewsTitle}>Reviews and Ratings</h3>
+              <div className={styles.overallRating}>{starRating}</div>
+              <div className={styles.starRating}>
+                <ReviewsAverageOverviewStars starRating={starRating} />
               </div>
-              {' '}
-              of reviewers recommend this product
-            </div>
-            <ReviewRatingDistribution
-              className={styles.ratingDistribution}
-              reviewCount={totalNumberOfStars}
-              fiveStarTotal={Number(fiveStarTotal)}
-              fourStarTotal={Number(fourStarTotal)}
-              threeStarTotal={Number(threeStarTotal)}
-              twoStarTotal={Number(twoStarTotal)}
-              oneStarTotal={Number(oneStarTotal)}
-            />
-            <div className={styles.reviewTotal}>
-              {totalNumberOfStars}
-              {' '}
-              Total Reviews
+              <div className={styles.recommendOverview}>
+                <div className={styles.recommendedPercent}>
+                  {recommendPercent}
+                  %
+                </div>
+                {' '}
+                of reviewers recommend this product
+              </div>
+              <ReviewRatingDistribution
```

Comment:
the same util we used to move starRatings out of App state can be applied here if you have time

Reply:
Ooo that's a good idea actually - will attempt in the morning! Definitely appreciate the feedback 😊